00-intro.md

```diff
@@ -64,13 +64,13 @@ Bash is the default shell on most modern implementations of Unix
 and in most packages that provide Unix-like tools for Windows.
 
 Using Bash or any other shell
-sometimes feels more like programming than like using a mouse.
```

Comment:
Right. But isn't saying "shell is more **like** programming" enough to make this distinction? I am not super strong about it, but I think it would be an improvement. If you disagree I can remove this change.